changelogs/changelog.yaml

```diff
@@ -1 +1,11 @@
-releases: {}
+ancestor: null
+releases:
+  0.1.0:
+    changes:
+      release_summary: 'Our first release matches the ``hashi_vault`` lookup functionality
+        provided by ``community.general`` ver ``1.3.0``.
```

Comment:
I think `version` should be written out.

Reply:
er, I do too, I must've gotten distracted in the middle of writing the word 🤦‍♂️

Reply:
Hehe, that does happen :)